extend.php

```diff
@@ -90,5 +93,25 @@
 
     (new Extend\Settings())
         ->serializeToForum('fof-discussion-language.composerLocaleDefault', 'fof-discussion-language.composerLocaleDefault', 'boolVal')
-        ->serializeToForum('fof-discussion-language.showAnyLangOpt', 'fof-discussion-language.showAnyLangOpt', 'boolVal'),
+        ->serializeToForum('fof-discussion-language.showAnyLangOpt', 'fof-discussion-language.showAnyLangOpt', 'boolVal')
+        ->serializeToForum('fof-discussion-language.useLocaleForTagsPageLastDiscussion', 'fof-discussion-language.useLocaleForTagsPageLastDiscussion', 'boolVal'),
+
+    (new Extend\Event())
+        ->listen(TagCreating::class, Listeners\TagCreating::class)
+        ->subscribe(Listener\UpdateTagMetadata::class),
+
+    (new Extend\ApiSerializer(TagSerializer::class))
+        ->attributes(function (TagSerializer $serializer, Tag $tag, array $attributes) {
+            $json = json_decode($tag->localised_last_discussion, true);
+
+            // Attach discussion title as this is needed for the tags page
+            foreach ($json as $languageId => $data) {
+                $data['title'] = optional(Discussion::find($data['id']))->title || '';
+                $json[$languageId] = $data;
+            }
+
+            $attributes['localisedLastDiscussion'] = $json;
+
+            return $attributes;
```

Comment:
I'd prefer to move this into an invokable class, so that we keep the `extend.php` a little tidier :)

Reply:
Done!